tests/onboarding/DeleteTest.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {test, expect} from '@playwright/test'
+import Homepage from '../../page-object/home/HomePage'
+import DeletePage from'../../page-object/onboarding/DeletePage'
+import SignupPage from '../../page-object/onboarding/SignupPage'
+import UtilityHelper from '../../utility/UtilityHelper'
+
+const testData = JSON.parse(JSON.stringify(require('../../test-data/test-data.json')))
+
+test('To delete account',async({page})=> {
```

Comment:
Description of the test should be `Test Delete Account` and there should be a space before =>

Reply:
fixed